### PR TITLE
Arcane tomes turn all water they touch into unholy water and/or blood.

### PR DESCRIPTION
**Pull request by harmonyn**
:cl: 
add: Arcane tomes turn all water they touch into unholy water and/or blood.
/:cl:

I thought this was a thing, but it wasn't .So: 
- Arcane tomes can turn all water they touch, such as water tanks, into unholy water, like the chaplain's blessing for cultists. Given that cultists can already smack each other with tomes to get the same healing effect, I don't see this as too bad a buff. 
- Non-cultists who get their hands on a tome, via emagging, theft, loot drops or admin shenanigans, can do the same to turn the water into blood. Useful for spookiness, gore and making cleaning 1% more difficult.
- Rewrote some tome messages to be more thematic/grammatical.

tomorrow I will fix the copypasta on this and bibles

### CODE REVIEW

**Comment by jughu** [comment]
But can the chaplain turn unholy water into holy water?

**Comment by Super3222** [comment]
Neat, now we don't need a Chaplain to confirm a cult.

**Comment by bgobandit** [comment]
Chaplains can purify unholy water, yes.

**Comment by ShadowDeath6** [comment]
so is there any reason to get the unholy water flask from the archives structure anymore

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -217,20 +217,20 @@ This file contains the arcane tome files.
 		if(ticker.mode.name == "cult")
 			var/datum/game_mode/cult/cult_mode = ticker.mode
 			if(!("eldergod" in cult_mode.cult_objectives))
-				user << "<span class='warning'>Nar-Sie does not wish to be summoned!</span>"
+				user << "<span class='warning'>\"This is not the time for the great summons,\" hisses a strange, hollow voice. \"Do not disturb me.\"</span>"
```
Comment (Xhuis) [inline code/game/gamemodes/cult/ritual.dm]: Nar-Sie/cult has its own span. Not sure which - check the style sheet.

**Comment by ShadowDeath6** [comment]
also while blood is cool, it seems kind of odd that every single cult item is either unusuable/unreadable or physically punishes noncult for using them but the tome can have its magic utilized with no negative penalty

**Comment by InsaneHyena** [comment]
Can we also make the unholy water look like blood when splashed? It would be very cool.

**Comment by optimumtact** [comment]
I'd like to merge this if you can finish it please

**Comment by KorPhaeron** [comment]
Let me know if/when you finish, like the idea

**Pull request closed without merging**